packages/Babylonian-UI.package/BPIconicButton.class/class/newHaltButtonOfSize.andExtentOffsetX..st

```diff
@@ -1,11 +1,11 @@
 instance creation
 newHaltButtonOfSize: aSize andExtentOffsetX: anOffset
 	
-	| exploreButton |
-	exploreButton := IconicButton new.
-	exploreButton 
+	|button|
+	button := IconicButton new.
+	^ button
```

Comment:
that line should probably be
`^ IconicButton new`
to make a proper cascade, you don't need to create a `|button|` variable

Reply:
We had that discussion before. I need to access the extent of that button in the cascade below. I can't access it via `self extent` or similar but have to store the instance beforehand instead.

Reply:
you're right, it was easy to miss